source/image-handler/bitmap-handler.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import Jimp from "jimp";
+import { ImageHandlerError, StatusCodes } from "./lib";
+
+export class BitmapRequest {
```

Comment:
This class mostly converter from BMP to PNG, IMO `ImageConverter` is more appropriate name, or you can get rid of it at all and move into `image-request.ts`, like

```ts
private convert(imageBuffer: Buffer): Promise<Buffer> {
  // if image is BMP return converted into PNG buffer
  // else do nothing and return `imageBuffer`
}